docs/merge-into.md

```diff
@@ -0,0 +1,35 @@
+** (docs merge) This extension integrates SEPA (Single Euro Payments Area) into CiviCRM. It's the perfect way to 
+manage direct debit payments in the whole EURO zone without having to resort to an expensive bureau.**
+
+
+## Status
+In the course of the development the project unfortunately split into three branches, mostly due to some very tight deadlines with different requirements. Since early July 2014 there is a joint version, merging the branches of Xavier Dutoit (tttp) and Björn Endres (SYSTOPIA). Both versions have, individually, been used in production environments since the beginning of 2014.
+
+Maintainer: Björn Endres (endres (at) systopia.de) and Xavier Dutoit (xavier (at) tttp.eu)
```

Comment:
I think we can remove Xavier as a maintainer, but he should still be credited for the initial development